ballista/scheduler/src/standalone.rs

```diff
@@ -38,6 +38,7 @@ pub async fn new_standalone_scheduler() -> Result<SocketAddr> {
             "localhost:50050".to_owned(),
             Arc::new(client),
             BallistaCodec::default(),
+            100000,
```

Comment:
Is it intentionaly to use `100,000` here and not `10,000` like other places?

Reply:
It's not intentional. It's better to set it to be the same as other places. I'll change it.